vue-client/src/resources/json/combinedTemplates.json

```diff
@@ -964,6 +964,200 @@
+              "mainTitle": "",
+              "subtitle": ""
+            }
+          ],
+          "responsibilityStatement": "",
+          "marc:primaryProvisionActivity": {
+            "@type": "PrimaryProduction",
+            "country": {
+              "@id": "https://id.kb.se/country/sw"
+            },
+            "marc:publicationStatus": "marc:QuestionableDate",
+            "otherYear": "",
+            "year": ""
+          },
+          "production": [
+            {
+              "@type": "Production",
+              "agent": {
+                "@type": "Agent",
+                "label": [
+                  ""
+                ]
+              },
+              "date": [
+                ""
+              ],
+              "place": [
+                {
+                  "@type": "Place",
+                  "label": [
+                    ""
+                  ]
+                }
+              ]
+            }
+          ],
+          "carrierType": [{"@id": "https://id.kb.se/term/rda/Volume"}],
+          "baseMaterial": [{"@id": "https://id.kb.se/term/rda/material/Paper"}],
+          "extent": [
+            {
+              "@type": "Extent",
+              "label": ""
+            }
+          ],
+          "physicalDetailsNote": "illustrationer",
+          "productionMethod": [
+            {
+              "@type": "ProductionMethod",
+              "label": [
+                "handskrift"
+              ]
+            }
+          ],
+          "hasDimensions": {
+            "@type": "Dimensions",
+            "label": ""
+          },
+          "hasNote": [
+            {
+              "@type": "Note",
+              "label": [
+                "Ingår i KB:s handskriftssamling"
+              ]
+            },
+            {
+              "@type": "Note",
+              "label": ""
+            }
+          ],
+          "instanceOf": {
+            "@type": "Text",
+            "language": [
+              {
+                "@id": "https://id.kb.se/language/swe"
+              }
+            ],
+            "genreForm": [
+              {
+                "@id": "https://id.kb.se/term/saogf/Handskrifter"
+              }
+            ],
+            "contribution": [
+              {
+                "@type": "PrimaryContribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/unspecifiedContributor"
```

Comment:
Pointless?

Reply:
Hmm maybe, I need to investigate what the actual use-case would be to specify unspecified. My suspicion is that it is used when there is only one contributor and it is not the author (which usually has been inferred when missing).

Reply:
There was nothing in the instructions regarding this, so I removed it.